lib/dndcolors.sty

```diff
@@ -0,0 +1,61 @@
+% Define colors, sampled from the books.
+\usepackage{color}
+
+% Page
+\definecolor{bgtan}{HTML}{F7F2E5}		  % background and quotebox
+\definecolor{pagegold}{HTML}{B89A67}  % pagenumbers and footer
+
+% Type
+\definecolor{titlered}{HTML}{58180D}	% titles
+\definecolor{titlegold}{HTML}{C9AD6A} % titlerules
+\definecolor{rulered}{HTML}{9C2B1B}		% triangular rule in statsblock
```

Comment:
These colors that specifically mention an actual color in their name could be changed now to more generic names:
```
\definecolor{DndTitleRed}{HTML}{58180D}
\colorlet{titlecolor}{DndTitleRed}
```
Although you have already done a lot of work on this, so we could push it off until later. If you are feeling "done" with this.

Reply:
While those colours should be "public" (i.e., documented), they don't change with the theme. I'd prefer to keep this PR scope limited to the theme colours.